media/css/pocket/components/updates-signup.scss

```diff
@@ -24,10 +30,95 @@ $image-path: '/media/protocol/img';
 
     p {
         @include text-body-lg;
+    }
 
-        a {
-            color: $color-action-primary;
+    .mzp-c-button {
+        background-color: $color-button-red;
+        border: 0;
+        height: 40px;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: color.adjust($color-button-red, $lightness: -10%);
+            color: $color-white;
         }
     }
+}
+
+    .pocket-updates-form {
+        .mzp-u-list-styled {
+            margin: 8px;
+            padding: 0;
+
+            li {
+                margin: 0;
+            }
+        }
+
+        fieldset {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+
+            label {
+                @include visually-hidden;
+            }
+
+            @media (min-width: 870px) {
```

Comment:
Issue: noticed a small visual issue at medium view ports where the field set pushes up against the image

![image](https://github.com/mozilla/bedrock/assets/400117/4215aab3-e3cc-41a4-b681-a513334c1bc4)

Reply:
Fixed now by pushing the breakpoint to 960px - looks ok to me, but second eyes welcome once I've updated demo (will comment on this PR when it's up again)